src/ir/cfg_builder.cpp

```
@@ -84,6 +84,11 @@ struct CfgBuilder final {
         }
         prog.m_assertions.insert_or_assign(label, assertions);
     }
+
+    void set_callback_metadata(std::set<int32_t> target_labels, std::set<int32_t> targets_with_exit) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'set_callback_metadata' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)